roles/contiv/tasks/netplugin_iptables.yml

```diff
@@ -23,7 +23,36 @@
   notify: Save iptables rules
 
 - name: Netplugin IPtables | Open vxlan port with iptables
-  command: /sbin/iptables -I INPUT 1 -p udp --dport 8472 -j ACCEPT -m comment --comment "vxlan"
+  command: /sbin/iptables -I INPUT 1 -p udp --dport 8472 -j ACCEPT -m comment --comment "netplugin vxlan 8472"
```

Comment:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/meta/main.yml#L21-L28 is an example of such use, just put it in the dependencies for this role like it's done there for the node role